src/pybind/mgr/balancer/module.py

```diff
@@ -740,7 +740,11 @@ def do_crush_compat(self, plan):
             self.log.error(detail)
             return -errno.EOPNOTSUPP, detail
 
-        key = 'pgs'  # pgs objects or bytes
+        # pgs, objects, or bytes
+        key = self.get_config('crush_compat_key', 'pgs')
```

Comment:
crush_compat_metric?

also we could make this accept a list, maybe, of one, two, or three of the keys?

Reply:
Done below. For now we still only rebalance with the first listed metric.